dbunit/src/test/java/com/baeldung/dbunit/OldSchoolDbUnitTest.java

```diff
@@ -0,0 +1,111 @@
+package com.baeldung.dbunit;
+
+import org.dbunit.Assertion;
+import org.dbunit.IDatabaseTester;
+import org.dbunit.JdbcDatabaseTester;
+import org.dbunit.dataset.IDataSet;
+import org.dbunit.dataset.ITable;
+import org.dbunit.dataset.xml.FlatXmlDataSet;
+import org.dbunit.operation.DatabaseOperation;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.ResultSet;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class OldSchoolDbUnitTest {
+    private static final String JDBC_DRIVER = org.h2.Driver.class.getName();
+    private static final String JDBC_URL = "jdbc:h2:mem:default;DB_CLOSE_DELAY=-1;init=runscript from 'classpath:schema.sql'";
+    private static final String USER = "sa";
+    private static final String PASSWORD = "";
+
+    private static IDatabaseTester tester = null;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        tester = initDatabaseTester();
+    }
+
+    private static IDatabaseTester initDatabaseTester() throws Exception {
+        final JdbcDatabaseTester tester = new JdbcDatabaseTester(JDBC_DRIVER, JDBC_URL, USER, PASSWORD);
```

Comment:
I would recommend against the use of `final` here - it's not wrong, but it doesn't add an extra meaning to the code, so is more to read.

The `private static final`s at the top of the file ARE conventional